Firestore/core/test/unit/model/field_path_test.cc

```diff
@@ -26,6 +28,12 @@ namespace firebase {
 namespace firestore {
 namespace model {
 
+namespace {
+FieldPath Parse(const std::string& path) {
+  return FieldPath::FromServerFormat(path).ConsumeValueOrDie();
+}
+}  // anonymous namespace
```

Comment:
Ultranit: the more common Google style convention is to just use a blank "name" for an unnamed namespace; i.e. this should be just `// namespace`, without the `anonymous` part.

Reply:
Done. I also fixed the instance I was inspired by.